apps/environment-variables-fetch-example/src/main.ts

```diff
@@ -1,9 +1,14 @@
 import { bootstrapApplication } from '@angular/platform-browser';
+import { EnvironmentVariable } from '@ngx-nuts-and-bolts/environment-variables-example-app-base';
 import { AppComponent } from './app/app.component';
 import appConfig from './app/app.config';
 
 fetch('./assets/env.json')
 	.then((response) => response.json())
-	.then((env) => {
+	.then((env: Record<EnvironmentVariable, string>) => {
 		bootstrapApplication(AppComponent, appConfig(env)).catch((err) => console.error(err));
+	})
+	.catch((e) => {
```

Comment:
There is no smart way of handling this error. If we can not fetch env.json, app will simply not load.

Reply:
But you added `throw e` anyway? For such case, I would add eslint-ignore comment.